tools/gulp/tasks/unit-test.ts

```diff
@@ -75,13 +75,15 @@ gulp.task('test', [':test:deps'], () => {
   });
 
   // Refreshes Karma's file list and schedules a test run.
```

Comment:
Expand comment to say something like "Only run tests if TypeScript compilation is successful" ?